docs/en/dataset_prepare.md

```diff
@@ -269,6 +281,35 @@ Since we only support test models on this dataset, you may only download [the va
 
 Since we only support test models on this dataset, you may only download [the test set](http://data.vision.ee.ethz.ch/daid/NighttimeDriving/NighttimeDrivingTest.zip).
 
+### A2D2
+
+To set up the A2D2 semantic segmentation dataset, first download 'Dataset - Semantic Segmentation' (`camera_lidar_semantic.tar`) from the official site ([a2d2.audi/a2d2/en/download.html](https://www.a2d2.audi/a2d2/en/download.html)). Extract the downloaded file as `a2d2/camera_lidar_semantic/`. Inside this directory, there will be several subdirectories representing data sequences. Each sequence directory contains `camera`, `label`, `lidar` subfolders.
+
+Next, while in the MMSegmentation directory root, create a symbolic link from `mmsegmentation/data/a2d2` to the dataset directory `a2d2/camera_lidar_semantic`.
+
+```shell
+ln -s /absolute/path/to/a2d2/camera_lidar_semantic/ data/a2d2/
+```
+
+Finally, convert the A2D2 dataset label files to the MMSegmentation format. One can choose to generate labels with a merged set of 18 classes (default choice). The official A2D2 paper presents benchmark results with the same classes and data split. (ref: p.8 "4. Experiment: Semantic segmentation"). The original A2D2 semantic segmentation category labels with 34 classes (reduced from 38 classes as explained below) by adding the option `--choice 34_cls` when running the conversion script. The original label files will not be overwritten during conversion. It is possible to have both 18 and 34 class labels co-existing in the same A2D2 data directory as the post-processed files do not overwrite each other.
```

Comment:
where is `ref: p.8 "4. Experiment: Semantic segmentation"` ?

Reply:
What do you mean? The sentence mentions 'The official A2D2 paper ', and refers to section "4. Experiment: Semantic segmentation" on p.8 in the paper.